src/table/control_codes.h

```diff
@@ -171,6 +171,7 @@ enum StringControlCode : uint16_t {
 	 * These are mapped to the original glyphs */
 	SCC_LESS_THAN        = SCC_SPRITE_START + 0x3C,
 	SCC_GREATER_THAN     = SCC_SPRITE_START + 0x3E,
+	SCC_LEFT_ARROW       = SCC_SPRITE_START + 0x7F,
```

Comment:
In NewGRFs 0x7F is occupied by `SCC_NEWGRF_PRINT_DWORD_CURRENCY`.
While `TranslateTTDPatchCodes` can map anything to anything, right now all the glyphs are mapped 1:1.

I would prefer to use 0x9D. And adding it to `TranslateTTDPatchCodes`.